components/Sidebar/ClientSidebar.tsx

```
@@ -248,110 +267,246 @@ function ClientSidebarContent({
             }
           } else if (data.type === "HEARTBEAT") {
             // Server heartbeat received
-          } else if (data.type === "chat.created") {
+          } else if (isEventType(EVENT_TYPES.CHAT_CREATED, "chat.created")) {
             // Handle chat creation - redirect to the chat page
             const chatData = data.data;
-            console.log("Chat created event received:", chatData);
 
             if (chatData && chatData.id) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)